### PR TITLE
Conduits, shuttle, random maps.

### PR DESCRIPTION
**Pull request by Zuhayr**
- Removed the crew transfer code/vote
- Reskinned the escape shuttle messages to an 'evacuation system'.
- Fixed a minor runtime with missing robotics manufacturers.
- Added icons/code for basic conduit powernets (no use yet).
- Added a fourth Z-level (surface), applies tundra random map to it.
- Fixes the seafloor random map.

image

image

### CODE REVIEW

**Comment by Finalsong** [comment]
"No, I haven't played SOMA recently, why do you ask?" - Zuhayr, 2015.

On a more serious note, awesome. I can't wait to see how the fourth z-level turns out.

**Comment by Loganbacca** [comment]
Europa surface?

**Comment by Zuhayr** [comment]
I'm going to resprite the trees into ice structures and natural features later, this will do for now to add some texture.

**Comment by Techhead0** [comment]
The trees are plastic, placed there by homesick colonists.

File: code/controllers/emergency_shuttle_controller.dm [hunk 1]
```diff
@@ -44,17 +34,7 @@ var/global/datum/emergency_shuttle_controller/emergency_shuttle
...
-				emergency_shuttle_docked.Announce("The Emergency Shuttle has docked with the station. You have approximately [round(estimate_launch_time()/60,1)] minutes to board the Emergency Shuttle.")
-			else
-				priority_announcement.Announce("The scheduled Crew Transfer Shuttle has docked with the station. It will depart in approximately [round(emergency_shuttle.estimate_launch_time()/60,1)] minutes.")
-
-		//arm the escape pods
-		if (evac)
-			for (var/datum/shuttle/ferry/escape_pod/pod in escape_pods)
-				if (pod.arming_controller)
-					pod.arming_controller.arm()
+			emergency_shuttle_docked.Announce("The emergency evacuation system is now prepared to fire. You have approximately [round(estimate_launch_time()/60,1)] minutes to board the evacuation pod.")
```
Comment (Loganbacca) [inline code/controllers/emergency_shuttle_controller.dm]: Will there be more than one evacuation pod?

Comment (Zuhayr) [inline code/controllers/emergency_shuttle_controller.dm]: Nope. Evacuating from an isolated colony should be difficult.

**Comment by Loganbacca** [comment]
Tested the shuttle in this PR's current state and from what I can tell it all worked well (aside from drowning upon arrival).

Issues I noticed:
 - Bridge communication console needs some rewording
 - Vote selection window options were a little broken

Seafloor generation is perhaps a little dense with seaweed. More "empty" seafloor with seaweed and plants clustered around man made structures and rock outcroppings would be ideal (source: I scuba dive IRL). 

The seafloor with glowing seaweed is eerily beautiful by the way:
seafloor
Definitely going to need more windows.

**Comment by Zuhayr** [comment]
Clarify what you mean by the vote being messed up?

**Pull request merged**